### PR TITLE
feat: add a new plugin pexels

### PR DESCRIPTION
**Pull request by LinkW77**
# Pull Request

## PR Description

Add a new plugin pexels.

## Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Performance enhancement
- [ ] Code refactor
- [ ] Documentation update
- [ ] Other, please describe:

## Checklist

Before submitting this PR, please make sure:

- [x] I have read the [CONTRIBUTING.md](../CONTRIBUTING.md) guidelines.
- [x] I have tested my changes locally to ensure they are effective.
- [x] I have updated the necessary documentation (if applicable).

### CODE REVIEW

File: plugin/bundles/pexels/plugins/video_search/plugin.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+class VideoSearch(PluginHandler):
+    async def execute(self, credentials: BundleCredentials, plugin_input: PluginInput) -> PluginOutput:
+        query: str = plugin_input.input_params.get("query")
+        orientation: str = plugin_input.input_params.get("orientation")
+        size: str = plugin_input.input_params.get("size")
+        locale: str = plugin_input.input_params.get("locale")
+        page: int = plugin_input.input_params.get("page")
+        per_page: int = plugin_input.input_params.get("per_page")
+
```
Comment (DynamesC) [inline plugin/bundles/pexels/plugins/video_search/plugin.py:18]: Add checks for input values, including orientation, size, per_page

**Review by DynamesC — APPROVED** [review]
LGTM

**Review by jameszyao — APPROVED** [review]
LGTM

**Review by jameszyao — APPROVED** [review]
LGTM

**Pull request merged**